packages/unified-latex-to-pretext/libs/report-and-expand-macros.ts

```diff
@@ -0,0 +1,42 @@
+import * as Ast from "@unified-latex/unified-latex-types";
+import { newcommandMatcher, newcommandMacroToSubstitutionAst } from "@unified-latex/unified-latex-util-macros";
+import { anyMacro, match } from "@unified-latex/unified-latex-util-match";
+import { visit } from "@unified-latex/unified-latex-util-visit";
+import { KATEX_SUPPORT } from "./pre-conversion-subs/katex-subs";
+import { replaceNode } from "@unified-latex/unified-latex-util-replace";
+
+/**
+ * Return list of macros unsupported by Katex
+ */
+export function report_macros(ast: Ast.Ast): string[] {
```

Comment:
JS tradition is to use camelCase names. Make a more specific name, like `reportMacrosUnsupportedByKatex`